app/user/userController.js

```diff
@@ -0,0 +1,84 @@
+import crypto from 'crypto'
+import passport from 'koa-passport'
+import LocalStrategy from 'passport-local'
+import User from './userModel'
+
+
+const authenticate = (type,ctx) =>{
+                                    return passport.authenticate(type,(err, user, info, status) => {
```

Comment:
you should install a linter to your code editor, to help with well formatted code. Check https://prettier.io/docs/en/editors.html